### PR TITLE
DEVDOCS-4301: Update handlebar-helpers-reference.md

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-4301](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4301)

## What changed?
I worked with Evan Johnson to update the `get` definition. He suggested that `getObject` and `option` could stay the same.  `moment` doesn't work with `concat` in the same way so this definition was not changed.

### CODE REVIEW

**Review by slsriehl — CHANGES_REQUESTED** [review]
see [slack convo](https://bigcommerce.slack.com/archives/C039JSWHH50/p1658173450826809) for suggested changes.

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ The following table contains BigCommerce's open source [Handlebars helpers](http
 | [pluck](#pluck) | array | Uses search key to get values from collections. |
 | [cdn](#cdn) | assets | A URL transformer for content delivery networks. |
 | [money](#money) | currency | Formats number length, thousands delimiter, and decimal delimiter. |
+| [moment](#moment) | date | Use [moment](https://momentjs.com/) as a helper. |
```
Comment (slsriehl) [inline docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:17]:
```suggestion
| [moment](#moment) | date | Use [momentjs](https://momentjs.com/) as a helper. |

```
Everywhere you're referring to the library, use `momentjs`. When you're referring to the date objects, `moment` is correct.

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md [hunk 2]
```diff
@@ -242,6 +246,29 @@ As highlighted above, the helper is configured to rewrite *local* URLs to an `as
...
+```handlebars
+{{moment date format datejs=false}}
+<!-- To disable date.js -->
+```
+
+```handlebars
+{{moment "January 1, 2022" format="YYYY-MM-DD"}}
+<!-- To call moment.js functions -->
+```
+
```
Comment (slsriehl) [inline docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:271]: are there any examples of calculations? for example, before we brought the helper in-house, this worked: `{{moment "5 hours ago" "MM/DD/YYYY HH:mm"}}` 

Can we check to see if it still does?

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md [hunk 3]
```diff
@@ -242,6 +246,29 @@ As highlighted above, the helper is configured to rewrite *local* URLs to an `as
...
 - [See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=cdn)
 
+
+### {{moment}}
+
+```handlebars
+{{moment date format}}
+```
+
+Use [moment](https://momentjs.com/) to format dates.
```
Comment (slsriehl) [inline docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:256]:
```suggestion
Use [momentjs](https://momentjs.com/) to format and calculate dates.

```

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md [hunk 4]
```diff
@@ -942,6 +969,27 @@ Renders a block if a statement is false; does not support operators for comparis
...
 - [See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=unless)
 
+
+### {{option}}
+
+```handlebars
+{{option "a.b.c"}}
+```
+Returns the given value of prop from `this.options`.
+
```
Comment (slsriehl) [inline docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:981]: add an example block with an object, for example (can't suggest a code block)

```
this.options = {
  a: {
    b: {
      c: "exampleValue"
    }
  }
}
```

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md [hunk 5]
```diff
@@ -942,6 +969,27 @@ Renders a block if a statement is false; does not support operators for comparis
...
+### {{get}}
+```handlebars
+{{get (concat “a.b” “.c.d”) someObject}}
+```
+Use property paths (`a.b.c`) to get a value or nested value from the context. Works as a regular helper or block helper. You can use return values from the `concat` helper as property paths.
```
Comment (slsriehl) [inline docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:986]:
```suggestion
Use property paths (`a.b.c.d`) to get a value or nested value from the context. Works as a regular helper or block helper. You can use return values from the `concat` helper as property paths.

```

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md (hunk 4 above)
Comment (slsriehl) [reply docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:981]: will this return a number/integer, or only a string?

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md (hunk 5 above)
Comment (traci707) [reply docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:986]: {{#get}} helper performs a “browse” query that creates a block of data that represents a list of your posts, authors, tags, or tiers.

**Review by markcmurphy — APPROVED** [review]
Approving since the GA date was last week and issues are addressed

**Pull request merged**